src-ui/list.html

```diff
@@ -5,6 +5,7 @@
 <meta http-equiv="Content-Type" content="text/html">
 <TITLE>puzz.link list of puzzle types</TITLE>
 <link rel="stylesheet" href="./css/list.css?<%= git.hash %>">
+<script src="./js/candle.js?<%= git.hash %>"></script>
```

Comment:
(this is more of a note to self / for later)

Feels wrong to load candle.js for the puzzle type list. It turns out that list.js only uses pzpr.js for

- pzpr.lang
- pzpr.variety

Besides getting rid of list.html entirely (or at least in its dynamic form), does it seem like the module stuff would allow us at some point to only selectively import those little parts of pzpr.js that we need from list.js?

Reply:
I expect so.